network_acl_sample.js

```diff
@@ -20,11 +20,9 @@ function _validateEnvironmentVariables() {
 }
 
 const util = require('util');
-const msRestAzure = require('ms-rest-azure');
-const KeyVault = require('azure-keyvault');
-const AuthenticationContext = require('adal-node').AuthenticationContext;
-const KeyVaultManagementClient = require('azure-arm-keyvault');
-const ResourceManagementClient = require('azure-arm-resource').ResourceManagementClient;
+const { ClientSecretCredential } = require("@azure/identity")
```

Comment:
Please change ClientSecretCredential to DefaultAzureCredential as service team's requirement. Thanks